src/components/IOUConfirmationList.js

```diff
@@ -134,46 +137,41 @@ class IOUConfirmationList extends Component {
         }];
 
         this.state = {
+            sections: this.getSections(formattedParticipants),
+            currentActor: formattedParticipants[0],
             participants: formattedParticipants,
+            selectedParticipants: formattedParticipants,
         };
 
+        this.allOptions = OptionsListUtils.flattenSections(this.state.sections);
+
         this.toggleOption = this.toggleOption.bind(this);
-        this.onPress = this.onPress.bind(this);
+        this.confirm = this.confirm.bind(this);
+        this.scrollToIndex = this.scrollToIndex.bind(this);
+        this.maybeToggleParticipant = this.maybeToggleParticipant.bind(this);
     }
 
     componentDidMount() {
         // We need to wait for the transition animation to end before focusing the TextInput,
         // otherwise the TextInput isn't animated correctly
         setTimeout(() => this.textInput.focus(), CONST.ANIMATED_TRANSITION);
-    }
-
-    /**
-     * @param {String} value
-     */
-    onPress(value) {
-        if (this.props.iouType === CONST.IOU.IOU_TYPE.SEND) {
-            Log.info(`[IOU] Sending money via: ${value}`);
-            this.props.onSendMoney(value);
-        } else {
-            Log.info(`[IOU] Requesting money via: ${value}`);
-            this.props.onConfirm(this.getSplits());
-        }
-    }
 
-    /**
-     * Get selected participants
-     * @returns {Array}
-     */
-    getSelectedParticipants() {
-        return _.filter(this.state.participants, participant => participant.selected);
+        const enterConfig = CONST.KEYBOARD_SHORTCUTS.ENTER;
+        const modifiers = KeyboardShortcut.getShortcutModifiers(['CTRL']);
+        this.unsubscribeCTRLEnter = KeyboardShortcut.subscribe(
+            enterConfig.shortcutKey,
+            this.confirm,
```

Comment:
We shouldn't allow the button to be pressed when it's disabled (eg: all participants unselected)
```
if (this.props.isDisabled || this.props.isLoading) {
    return;
}
```

Reply:
I tried this (unselecting all participants during IOU confirmation), and pressing enter did not create the IOU. So I think we're good here?

Reply:
Case 1: Split bill from group, and unselect all participants in IOU confirmation page.
-> Pressing cmd + enter creates an IOU. This is an edge case which should be handled by the server, and once that's done I think it's okay to delegate it to the server and just show a error message.


https://user-images.githubusercontent.com/29673073/156629038-08a976c5-4f70-40fe-9861-267ea2cbda01.mov



Case 2: Split bill from global menu (FAB menu), and unselect all participants in IOU confirmation page. 
-> Pressing cmd + enter didn't create a IOU. So yeah, we're good for this case.

Reply:
> Pressing cmd + enter [without any selected participants] creates an IOU. This is an edge case which should be handled by the server, and once that's done I think it's okay to delegate it to the server and just show an error message.

I agree that this would be better error handling, but I prevented this on the front-end in the meantime. Created an internal issue to throw an error in the back-end if someone tries to do this: https://github.com/Expensify/Expensify/issues/205783

Reply:
Added testing steps for this case.